docs/azure-quantum/access.md

```diff
@@ -0,0 +1,23 @@
+# Setting up access and authentication to Azure Quantum
+
+[Azure Quantum](https://docs.microsoft.com/azure/quantum/overview-azure-quantum) is Microsoft's cloud service for running quantum computing programs or solving optimization problems and is currently in [Public Preview](https://cloudblogs.microsoft.com/quantum/2021/02/01/azure-quantum-preview/). You can try Azure Quantum for free today and submit quantum programs to our quantum partners and technologies. Follow the simple steps below to set up access and start sending quantum programs with Cirq to IonQ or Honeywell.
+
```

Comment:
As mentioned in the call, it might be good to call out when to use azure-quantum versus cirq-ionq to access ionq services.

Reply:
The addition for IonQ API key is good, but I might even expand it a bit for clarity.  For example:

`Note that there are two ways to access IonQ devices through cirq: using azure-quantum or by using cirq-ionq.  If you already have an Azure account or would like to request an IonQ key through Azure, use azure-quantum by following the steps in this tutorial.  If you already have a preexisting API key through other means, follow the steps outlined in [Cirq with the IonQ API](../ionq/access.md) to use the cirq-ionq package instead.`

Reply:
Thanks, great suggestion! Added a slightly modified version.